File: 2016 czerwiec/c++/1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("liczby.txt");
    ofstream out("wyniki_6_1.txt");

    int ans = 0;
    for (int i = 1; i <= 999; i++)
    {
        string s;
        in >> s;
        if (s[s.size() - 1] == '8')
            ans++;
    }
    out << ans;
}

File: 2016 czerwiec/python/1.py
with open("liczby.txt", "r") as f:
    file = f.read().strip().split('\n')

ans = 0
with open("wyniki_6_1.txt", "w") as f:
    for s in file:
        if s[-1] == '8':
            ans += 1
    f.write(str(ans))